app.js

```diff
@@ -0,0 +1,42 @@
+var express = require('express');
+var request = require('request');
+var bodyParser = require('body-parser');
```

Comment:
body-parser is available on the express itself. We can get rid of that dependency. 
Just change
`app.use(bodyParser.json());` -> `app.use(express.json());`
`app.use(bodyParser.urlencoded({extended: false}));` -> `app.use(express.urlencoded({extended: false}));`